### PR TITLE
DevDB: Version History and Listing Flagged Jobs

### PR DESCRIPTION
**Pull request by abrieff**
Includes both the version history report and the listing flagged jobs report.

### CODE REVIEW

**Comment by Oysters1874** [comment]
The report in general looks good to me. But still in terms of readability, probably we can think about how to make the description more straightforward. Also, is it possible to just display a list of job_numbers instead of a vertical long single-column dataframe, or do we need to add more fields into the dataframe to make it contain more information for eyeballing.

**Comment by abrieff** [comment]
Good q @Oysters1874 - my first thought was around what additional fields to include (for some of them, it might be nice to be able to access the relevant offending fields), but agreed that it should probably either be a list or include more columns

File: src/devdb/components/qaqc_app_report.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import streamlit as st
+
+
+class QAQCAppReport:
```
Comment (SashaWeinstein) [inline src/devdb/components/qaqc_app_report.py:4]: This could be renamed `ListFlaggedJobs` or something like that

**Review by SashaWeinstein — COMMENTED** [review]
One wordsmithing comment, thought it made sense to wait for PR #158 to be merged before I really dug in

File: src/devdb/devdb.py [hunk 2]
```diff
@@ -5,8 +5,32 @@ def devdb():
...
+
+    st.title("Developments Database QAQC")
+    st.markdown(
+        body="""
+        ### ABOUT DEVELOPMENTS DATABASE
+
+        DCP’s Developments Database is the agency’s foundational dataset used for tracking construction and providing high accuracy estimates of current and near-term housing growth and analysis of housing trends. 
+        It is the only source that can determine past and future change in Class A (permanent occupancy) and Class B (transient occupancy) housing units. 
+
+        The NYC Department of City Planning’s (DCP) Housing Database Project-Level Files contain all NYC Department of Buildings (DOB)-approved housing construction and demolition jobs filed or completed in NYC since January 1, 2010.
```
Comment (mbh329) [inline src/devdb/devdb.py:18]: Nice! Good to reuse the intro already in the wiki

**Comment by mbh329** [comment]
Agree with @SashaWeinstein. I think we can work on wordsmithing once the #158 is merged in

**Comment by td928** [comment]
not sure why I got a merge conflict when I pull the new changes from this branch. @abrieff is this a surprise to you at all?

**Comment by abrieff** [comment]
Oh not surprising cause I rebased on dev
Try git reset --hard origin head^ on the branch locally

**Comment by abrieff** [comment]
i'll just make a new branch to make this easier for people

**Pull request closed without merging**